src/cljs/main/broadfcui/common/table_utils.cljs

```diff
@@ -159,8 +159,8 @@
                                           default-render)]
                      (render-cell
                        {:width (:width col)
-                        :content (render-content (nth row (:declared-index col)))
-                        :title (render-title (nth row (:declared-index col)))
+                        :content (render-content (get row (:declared-index col)))
+                        :title (render-title (get row (:declared-index col)))
```

Comment:
This is the ugly hack to get this working.  Ultimately, the data ends up correct, but there's some kind of race condition where the table columns are getting updated while the data is still old, causing these `nth`s to throw an index-out-of-bounds error.  Switching to `get` suppresses the error, but doesn't actually fix the underlying problem.